nncf/experimental/torch/fx/transformations.py

```diff
@@ -528,6 +542,128 @@ def fuse_conv_bn(model: torch.fx.GraphModule) -> None:
     model.recompile()
 
 
+def _get_pattern_replacement_per_channel():
+    def pattern_per_channel(weight, scale, zero_point, axis, low, high, dtype):
```

Comment:
I see you support for zero_point == 0. Could you add support for asymmetric quantization as well?